bundles/mapping/mapmodule/plugin/getinfo/GetInfoPlugin.js

```diff
@@ -300,10 +300,16 @@ Oskari.clazz.define(
             const me = this;
             const dteMs = (new Date()).getTime();
             const requestedLayers = layers || me.getSandbox().findAllSelectedMapLayers();
-            const layerIds = me._buildLayerIdList(requestedLayers);
+            let layerIds = me._buildLayerIdList(requestedLayers);
```

Comment:
To make it easier to process here we might want to return the layers instead of their ids so we can filter again with the swipestatus/partial visibility info from the layer.